packages/utils/src/stacktrace.ts

```diff
@@ -25,7 +28,7 @@ export function createStackParser(...parsers: StackLineParser[]): StackParser {
 
       // https://github.com/getsentry/sentry-javascript/issues/5459
       // Remove webpack (error: *) wrappers
-      const cleanedLine = line.replace(/\(error: (.*)\)/, '$1');
+      const cleanedLine = /\(error: (.*)\)/.test(line) ? line.replace(/\(error: (.*)\)/, '$1') : line;
```

Comment:
Funny enough but this was actually quite a significant speedup as we are now no longer reallocating a new string for each line